plugins/nodenv/nodenv.plugin.zsh

```diff
@@ -0,0 +1,38 @@
+# This plugin loads nodenv into the current shell and provides prompt info via
+# the 'nodenv_prompt_info' function.
+
+FOUND_NODENV=$+commands[nodenv]
+
+if [[ $FOUND_NODENV -ne 1 ]]; then
+    nodenvdirs=("$HOME/.nodenv" "/usr/local/nodenv" "/opt/nodenv" "/usr/local/opt/nodenv")
+    for dir in $nodenvdirs; do
+        if [[ -d $dir/bin ]]; then
+            export PATH="$PATH:$dir/bin"
```

Comment:
```suggestion
            export PATH="$dir/bin:$PATH"
```

Reply:
Is there a specific reason why the nodenv folder must be first on `$PATH`?

According to [1] the required one for this is the shims folder, and I understand that this is taken care of by `nodenv init`.

[1]: https://github.com/nodenv/nodenv#understanding-shims

Reply:
If there is no other reason I have deferred to appending the directory then. There isn't much difference, but it is slightly more secure keeping the usual directories first in `$PATH`. Thanks for the review!

Reply:
Sounds good @mcornella. You might want to look at those lines in the rbenv plugin and make the adjustment if there is a flaw in how the `PATH` modification is done.